Zend/tests/named_params/unknown_named_param.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Unknown named parameter
+--FILE--
+<?php
+
+function test($a) {
+}
+
+function test2(...$a) {
+}
+
+try {
+    test(b: 42);
+} catch (Error $e) {
+    echo $e->getMessage(), "\n";
+}
+
+try {
+    test2(a: 42);
```

Comment:
This test indicates to me that in `test2(...$a)` when we pass in `test2(a: 42)` that a is _known_, but if you var_dump `$a` you don't get `[42]` you get `['a' => 42]`. That part doesn't make sense to me. It should just be `[42]`.